### ISSUE TITLE
undefined: ora.SetDrvCfg

### ISSUE THREAD

**Issue by asambeka** [rana/ora#46]
I am facing issues using SetDrvCfg method. Following example code provided in Readme.md fails to compile,

func init() {
    drvCfg := ora.NewDrvCfg()
    drvCfg.Env.StmtCfg.FalseRune = 'N'
    drvCfg.Env.StmtCfg.TrueRune = 'Y'
    drvCfg.Env.StmtCfg.Rset.TrueRune = 'Y'
    ora.SetDrvCfg(drvCfg)
}

Please suggest.

**Comment by tgulacsi** [comment]
Works For Me (tm) - which commit do you have at $GOPATH/src/gopkg.in/rana/ora.v3 ? Try with the latest (223b04c).
![2015-10-22-070424_1366x768_scrot](https://cloud.githubusercontent.com/assets/256074/10657425/73c200f2-788b-11e5-8b9d-b71eb758f207.png)

**Comment by asambeka** [comment]
I was using the master. I changed the setDrvCfg() call to Register() as per the documentation in Fix-Ora-24435, and this seems to have resolved. Looks like 223b04c is the latest fix you've made. Apparently when I did go get, I pulled previous version. I'll pull the 223b04c changes and will revert to serDrvCfg() call.
I must say I really liked this driver. I needed some solution to have custom row pre-fetch and this provided many more options (I was using oci8 before this and it has to way to configure that). I was able to pull 10Million records in under 5 mins with custom pre-fetch. It used to take over 20 mins with oci8.

Thanks.
